pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -63,16 +63,18 @@ Written comments in response to curricular/educational prompts within Code.org c
 Additional\* data stored by Code.org if you create a Code.org teacher account  | Required? | How this data is used
 --- | --- | ---
 Email address | Required | To send emails to the teacher with updates about their classroom or student progress, when new course-work is available, updates on curriculum, tools, professional learning options, etc.<br><br>All emails sent by Code.org will contain an unsubscribe link, and will not require typing a password to unsubscribe.
-District and School name and/or school type (Private, Public, Charter) | Optional | At your option and under your control, we would list your school in the Code.org map and database of schools that teach computer science courses.<br><br>Code.org may also use this information to reach out to your school or district to discuss broader education partnerships or participation in special events.
+District and School name and/or school type (Private, Public, Charter) | Optional | At your option and under your control, we would list your school in the Code.org map and database of schools that teach computer science courses.<br><br>Code.org or our Regional Partners\*\* may also use this information to reach out to your school or district to discuss broader education partnerships or participation in special events.
 School address | Optional | At your option and under your control, we would list your school (and relevant contact information, if you provide it) in the Code.org map and database of schools that teach computer science courses.
 Student section data | Required | You may create accounts for your students (and provide each student’s display name and optionally their age and gender), and organize these students into sections or groups. You may assign each section a display name, and a course assignment. The section grouping data is used to simplify your view of students across multiple sections.<br><br>Teachers are encouraged to share a document with students and parents informing them about the Code.org course, including the privacy implications.
-Survey and demographic data | Optional | For the purposes of evaluating our own work and improving our education results, Code.org regularly sends surveys to teachers. <br><br>These surveys are completely optional. The data provided by teachers in these surveys is saved and used for analysis by Code.org or by our Evaluator or our professional learning partners or facilitators. Any survey data shared with external parties will be anonymous and aggregated.
+Survey and demographic data | Optional | For the purposes of evaluating our own work and improving our education results, Code.org regularly sends surveys to teachers. <br><br>These surveys are completely optional. The data provided by teachers in these surveys is saved and used for analysis by Code.org or by our Evaluator or our Regional Partners or facilitators. Any survey data shared with external parties will be de-identified and aggregated.
```

Comment:
Should this be a list now? And I've heard that we should always put Code.org in front of Facilitators and Regional Partners for official docs?

`for analysis by Code.org, Code.org's Evaluator, Code.org Regional Partners, or Code.org Facilitators.`

Reply:
"Regional Partner" has a definition in our privacy policy now so I think we should leave it like that. We also say simply "facilitators" in the policy, so I think just keeping it like that is probably also fine. I am updating to make it more like a list like you suggested now though.